Assets/StreamingAssets/LUA/Furniture.lua

```diff
@@ -563,7 +563,7 @@ function OxygenCompressor_OnUpdate(furniture, deltaTime)
 end
 
 function OxygenCompressor_GetSpriteName(furniture)
-    baseName = "Oxygen_Compressor"
+    baseName = "compressor_oxygen"
```

Comment:
@abackwood Here you should get the name of the furniture programmatically instead of having it hardcoded.